projects/ngrx.io/content/marketing/presskit.md

```diff
@@ -0,0 +1,21 @@
+<h1 class="title center no-toc">Press Kit</h1><br />
+The logo graphics available for download on this page are provided under <a href="https://creativecommons.org/licenses/by/4.0/" target="_blank">CC BY 4.0</a>.
+
+## NgRx Logo
+<img src="assets/images/ngrx-badge.png" width="100" style="float:left; padding-right: 1rem;" />
+<div style="padding-top:1rem;">
+Logo (png) - <a href="#">Download</a><br />
```

Comment:
We have to add `href` to the logo here, otherwise it won't download.
The same for the other logos.